src/Build.UnitTests/BackEnd/BuildManager_Tests.cs

```diff
@@ -1598,7 +1598,7 @@ public void CancelledBuildWithDelay20()
             _logger.AssertLogDoesntContain("[errormessage]");
         }
 
-#if FEATURE_TASKHOST && !NO_MSBUILDTASKHOST
+#if !NO_MSBUILDTASKHOST
```

Comment:
Is the comment below still accurate? I'm wondering when we wouldn't have task hosts available.

Reply:
We don't have the 3.5 task host on .NET Core or Mono builds.